api/api.go

```diff
@@ -52,6 +52,7 @@ func Setup(router *mux.Router,
 	v1 := router.PathPrefix("/{version:v1}").Subrouter()
 	v1.HandleFunc("/search-reindex-jobs", api.GetJobsHandler).Methods("GET")
 	v1.HandleFunc("/search-reindex-jobs", api.CreateJobHandler).Methods("POST")
+	v1.HandleFunc("/search-reindex-jobs/{job_id}/tasks/{task_name}/number-of-documents/{count}", permissions.Require(update, api.PutNumOfDocsHandler)).Methods("PUT")
```

Comment:
I would be more explicit with the handler name

```suggestion
	v1.HandleFunc("/search-reindex-jobs/{job_id}/tasks/{task_name}/number-of-documents/{count}", permissions.Require(update, api.PutTaskNumOfDocsHandler)).Methods("PUT")
```

Reply:
Fixed.